src/components/common/Footer.js

```diff
@@ -6,8 +6,39 @@ const { Footer } = Layout;
 export default function MainFooter({ props }) {
   return (
     <div>
-      <Footer style={{ textAlign: 'center' }}>
-        Ant Design ©2018 Created by Ant UED
+      <Footer id="footer">
+        <div class="footer-container">
+          <div class="footer-section">
+            <h2>Contact Us</h2>
+            <a href="#.">email@email.com</a>
+            <a href="#.">Find a Program</a>
+          </div>
+
+          <div class="footer-section">
+            <h2>Services</h2>
+            <a href="#.">FAQ</a>
+            <a href="#.">Site Map</a>
+          </div>
+
+          <div class="footer-section">
+            <h2>Information</h2>
+            <a href="#.">About Coderheroes</a>
+            <a href="#.">Privacy Policy</a>
+            <a href="#.">Terms and Conditions</a>
+            <a href="#.">Press Inquiries</a>
+          </div>
+
+          <div style={{ textAlign: 'center' }} class="footer-section">
+            <h2>Coderheroes</h2>
+            <div class="social-links">
+              <a href="#.">Icon</a>
+              <a href="#.">Icon</a>
+              <a href="#.">Icon</a>
+              <a href="#.">Icon</a>
+            </div>
+            <small>Copyright &#169; 2020 coderheroes Inc.</small>
```

Comment:
@ashtilawat23 Hey, I'm personally curious about this <small> tag. Is it acceptable practice, should we use CSS?

Reply:
@ryan-hamblin ^^Engineering standards opinion?